packages/wallets/src/features/cashier/modules/WithdrawalCrypto/components/WithdrawalCryptoForm/WithdrawalCryptoForm.tsx

```diff
@@ -0,0 +1,126 @@
+                return (
+                    <form autoComplete='off' className='wallets-withdrawal-crypto-form' onSubmit={handleSubmit}>
+                        <div className='wallets-withdrawal-crypto-address'>
+                            <Field name='cryptoAddress' validate={validateCryptoAddress}>
+                                {({ field }: FieldProps<string>) => (
+                                    <WalletTextField
+                                        {...field}
+                                        helperMessage={errors.cryptoAddress}
+                                        label='Your BTC Wallet address'
+                                        showMessage
+                                    />
+                                )}
+                            </Field>
+                        </div>
+                        <WalletsPercentageSelector
+                            amount={
+                                !Number.isNaN(parseFloat(values.cryptoAmount)) ? parseFloat(values.cryptoAmount) : 0
+                            }
+                            balance={activeWallet?.balance || 0}
+                            onChangePercentage={percentage => {
+                                const fraction = percentage / 100;
+
+                                return setValues({
+                                    ...values,
+                                    cryptoAmount:
+                                        !!fraction && activeWallet?.balance
+                                            ? (fraction * activeWallet?.balance).toFixed(FRACTIONAL_DIGITS_CRYPTO)
+                                            : '',
+                                    fiatAmount:
+                                        !!fraction &&
+                                        activeWallet?.balance &&
+                                        activeWallet?.currency &&
+                                        exchangeRate?.rates
+                                            ? (
+                                                  (fraction * activeWallet?.balance) /
+                                                  exchangeRate?.rates[activeWallet.currency]
+                                              ).toFixed(FRACTIONAL_DIGITS_FIAT)
+                                            : '',
+                                });
+                            }}
+                        />
+                        <WithdrawalCryptoAmountConverter
+                            activeWallet={activeWallet}
+                            exchangeRate={exchangeRate}
+                            getCurrencyConfig={getConfig}
+                        />
+                        <div className='wallets-withdrawal-crypto-form__submit'>
+                            <WalletButton
+                                disabled={Object.keys(errors).length !== 0 || !values.cryptoAmount || isSubmitting}
```

Comment:
Disabling the `Withdraw` (submit) button to if there are any errors or withdrawal-amount is not entered or the form is still submitting.